src/content/docs/crowdin/translation-process/ai-setup-wizard.mdx

```diff
@@ -0,0 +1,106 @@
+---
+title: AI Setup Wizard
+description: Quickly configure AI features in Crowdin using the AI Setup Wizard
+slug: ai-setup-wizard
+sidebar:
+  order: 2
+---
+
+import { Image } from 'astro:assets';
+import { Steps, Aside, LinkCard, CardGrid } from '@astrojs/starlight/components';
+import ReadMore from '~/components/ReadMore.astro';
+import { Icon } from 'astro-icon/components';
+import QuestionAnswer from '~/components/QuestionAnswer.astro';
+import features from '!/crowdin/translation-process/ai_setup_wizard_features.png';
+import providers from '!/crowdin/translation-process/ai_setup_wizard_providers.png';
+import summary from '!/crowdin/translation-process/ai_setup_wizard_summary.png';
+
+The AI Setup Wizard simplifies and speeds up the configuration of Crowdin AI, allowing you to leverage AI-powered translations that adapt to various levels of context. With just a few steps, you can enable AI Pre-translate to apply these translations to your content or use AI as an assistant in the Editor for translation, proofreading, and more.
+
+Designed to get you started quickly, the wizard guides you through selecting needed features, configuring providers, and automatically creating AI prompts for the selected features.
+
+<ReadMore>
+  To further customize settings, read more about [configuring AI prompts](/crowdin-ai/#configuring-ai).
+</ReadMore>
+
+## When to Use the AI Setup Wizard
+
+You’ll see a **Start Using Crowdin AI Right Now!** banner if you haven’t configured your prompts yet. Click the **AI Setup Wizard** button to begin configuring Crowdin AI.
+
+If you’ve dismissed the banner, you can still access the wizard using the following URL (replace `{your-crowdin-username}` with your actual Crowdin username):
+
+```
+https://crowdin.com/profile/{your-crowdin-username}/ai/wizard
+```
```

Comment:
This is not so important and even distracts, I would suggest moving it to the Q&A section